scala/private/phases/phase_coverage.bzl

```diff
@@ -0,0 +1,78 @@
+#
+# PHASE: coverage
+#
+# DOCUMENT THIS
+#
+
+load(
+    "@io_bazel_rules_scala//scala/private:coverage_replacements_provider.bzl",
+    _coverage_replacements_provider = "coverage_replacements_provider",
+)
+
+def phase_coverage_library(ctx, p):
+    args = struct(
+        srcjars = p.collect_srcjars,
+    )
+    return _phase_coverage_default(ctx, p, args)
+
+def phase_coverage_common(ctx, p):
+    return _phase_coverage_default(ctx, p)
+
+def _phase_coverage_default(ctx, p, _args = struct()):
+    return _phase_coverage(
+        ctx,
+        p,
+        _args.srcjars if hasattr(_args, "srcjars") else depset(),
```

Comment:
this sounds like a bug in rules_scala that we don't support srcjars for binary rules.
In any case this is in no way a call for action on this PR. Just an observation.